src/GitTfs/Commands/RemoteOptions.cs

```diff
@@ -20,6 +20,8 @@ public OptionSet OptionSet
                         v => Username = v },
                     { "p|password=", "TFS password",
                         v => Password = v },
+                    { "no-parallel", "Do not access tfs in parallel",
```

Comment:
I think I prefer "Do not do parallel requests to TFS". What do you think?

Reply:
Agreed.